modules/administration/pages/tshoot-hostname-rename.adoc

```diff
@@ -49,25 +49,28 @@ The only mandatory parameter for the script is the newly configured IP address o
     If the server is not using the new hostname, the script fails.
     Be aware that this script refreshes the pillar data for all Salt minion: the time it takes to run depends on the number of registered Salt systems.
 . Re-configure your clients to make your environment aware of the new hostname and IP address.
-+
-In the Salt minion configuration file [path]``/etc/salt/minion``, you must make sure to specify the name of the new Salt master ({productname} Server):
+    In the Salt minion configuration file [path]``/etc/salt/minion``, you must make sure to specify the name of the new Salt master ({productname} Server):
 +
 ----
 master: <new_hostname>
 ----
-
-Once this is changed, restart the [literal]``salt-minion` process:
-
++
+. Once this is changed, restart the [literal]``salt-minion` process:
```

Comment:
```suggestion

. Restart the [literal]``salt-minion`` process:
```
If this rightly now is a step of its own, the `+` connector isn't needed.
And we we can drop the verbose intro or recap of the previous step ("Once this is...").